server/chat-plugins/poll.ts

```diff
@@ -454,9 +515,36 @@ export const commands: ChatCommands = {
 		`/poll results - Shows the results of the poll without voting. NOTE: you can't go back and vote after using this.`,
 		`/poll display - Displays the poll`,
 		`/poll end - Ends a poll and displays the results. Requires: % @ # & ~`,
+		`/deletequeue [number] - deletes poll with corresponding number from the queue.`,
+		`/clearqueue - deletes the queue of polls. Requires: % @ # & ~`,
+		`/viewqueue - view the queue of polls in the room. Requires: % @ # & ~`,
 	],
 };
 
+export const pages: PageTable = {
+	pollqueue(args, user) {
+		this.extractRoom();
+		const room = Rooms.get(args[0]) as ChatRoom | GameRoom;
+		let buf = `<div class = "pad"><strong>Queued polls</strong>`;
+		buf += `<button class="button" name="send" value="/join view-pollqueue-${room.roomid}" style="float: right">`;
+		buf += `<i class="fa fa-refresh"></i> Refresh</button><br />`;
+		if (!room.queuedActivity!.length) {
+			buf += `<hr/ ><strong>No polls queued.</strong></div>`;
+			return buf;
+		}
+		for (const poll of room.queuedActivity!) {
+			const num = room.queuedActivity?.indexOf(poll);
+			const button = (
+				`<button class="button" name="send" value="/poll deletequeue ${num},${room.roomid}">` +
```

Comment:
This does not seem to update the room upon clicking.

Reply:
that's what the refresh button is for, yeah. 
Is there a way to make the button send two commands?

Reply:
or, oh, i suppose it could use `onclick` and just run this.parse() twice?

Reply:
Ideally we could make the button send from a room?

In the meantime you can just flag it in the command. `/poll deletequeue num, roomid, updatelist`

Reply:
Okay.